sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,61 @@
+<div class="section no-pad-bot" id="splash-banner">
+  <div class="container">
+    <div class="row center">
+    <div class="brand-logo">
+      <img src="static/img/logotext.svg" alt="Sourcemash">
+    </div>
+    <h4 class="header col s12 light center">Read Your News. Faster.</h4>
+    </div>
+    <div class="row center">
+      <div class="btn btn-large waves-effect waves-light" id="need-account">Sign Up!</div>
+    </div>
+    <br>
+  </div>
+  <div id="splash-divider"></div>
+</div>
+<div class="section no-pad-bot" id="splash-info">
+  <div class="container">
+    <div class="row">
+
+      <h3 class="col s12 light center header flow-text" id="splash-subtitle">Sourcemash shortens the time it takes to read your daily news.</h3>
```

Comment:
I think this should be specific to categories. Like, "Sourcemash categorizes your articles to help you get through your feeds faster."